api/soap/mc_api.php

```diff
@@ -35,29 +35,67 @@
 require_api( 'api_token_api.php' );
 
 /**
+ * A class to capture a RestFault
+ */
+class RestFault {
+	/**
+	 * @var integer The http status code
+	 */
+	public $status_code;
+
+	/**
+	 * @var string The http status string
+	 */
+	public $fault_string;
+
+	/**
+	 * RestFault constructor.
+	 *
+	 * @param integer $p_status_code The http status code
+	 * @param string $p_faultstring The error description
+	 */
+	function __construct( $p_status_code, $p_fault_string = '' ) {
+		$this->status_code = $p_status_code;
+		$this->fault_string = $p_fault_string === null ? '' : $p_fault_string;
+	}
+}
+
+/**
```

Comment:
Will defer auto-loader work out this of review.  I would like to think it through along with wider composer usage.